c2corg_ui/static/js/documentediting.js

```diff
@@ -595,6 +594,46 @@ app.DocumentEditingController.prototype.updateMaxSteps = function(waypointType)
 
 
 /**
+ * Check what properties are missing and tell the user.
+ * @param {Object} doc
+ * @export
+ */
+app.DocumentEditingController.prototype.checkMissingProps = function(doc) {
+  var type = doc.type ? app.utils.getDoctype(doc.type) : this.modelName_ + 's'; // missing 's' in modelName if creating a new document
```

Comment:
> this.modelName_ + 's'; // missing 's' in modelName if creating a new document

What a hack :P 

You should rather use ``this.module_`` which contains the correct type name with the "s".
See https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/js/documentediting.js#L91
https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/waypoint/edit.html#L28